.eslintrc.yml

```diff
@@ -0,0 +1,20 @@
+env:
+  browser: true
+  es2021: true
+extends:
+  - 'plugin:react/recommended'
```

Comment:
> I think it doesnt need to be that complicated, next offers eslint with a prettier integration without much configuration effort : [See here](https://nextjs.org/docs/basic-features/eslint)

This uses the `eslint`-configuration from the Linkfree repo (which is a cra-app) mixed with some next-configuration right?

Reply:
Yes! But we still need the react plugin right? Because we are working with JSX?